modules/cnf-running-the-performance-creator-profile-offline.adoc

```diff
@@ -159,129 +184,102 @@ Flags:
-====
-
-. Check the machine config pool:
-+
-[source,terminal]
-----
-$ oc get mcp
-----
+* `-t /<path_to_must_gather_dir>/must-gather.tar.gz` specifies the path to directroy containing the must-gather tarball. This is a required argument for the wrapper script.
 +
 .Example output
-
 [source,terminal]
 ----
-NAME         CONFIG                                                 UPDATED   UPDATING   DEGRADED   MACHINECOUNT   READYMACHINECOUNT   UPDATEDMACHINECOUNT   DEGRADEDMACHINECOUNT   AGE
-master       rendered-master-acd1358917e9f98cbdb599aea622d78b       True      False      False      3              3                   3                     0                      22h
-worker-cnf   rendered-worker-cnf-1d871ac76e1951d32b2fe92369879826   False     True       False      2              1                   1                     0                      22h
+level=info msg="Cluster info:"
+level=info msg="MCP 'master' nodes:"
+level=info msg=---
+level=info msg="MCP 'worker' nodes:"
+level=info msg="Node: host.example.com (NUMA cells: 1, HT: true)"
+level=info msg="NUMA cell 0 : [0 1 2 3]"
+level=info msg="CPU(s): 4"
+level=info msg="Node: host1.example.com (NUMA cells: 1, HT: true)"
+level=info msg="NUMA cell 0 : [0 1 2 3]"
+level=info msg="CPU(s): 4"
+level=info msg=---
+level=info msg="MCP 'worker-cnf' nodes:"
+level=info msg="Node: host2.example.com (NUMA cells: 1, HT: true)"
+level=info msg="NUMA cell 0 : [0 1 2 3]"
+level=info msg="CPU(s): 4"
+level=info msg=---
 ----
 
-. Create a performance profile:
+. Create a performance profile by running the following command. This example uses sample PPC arguments and values:
 +
 [source,terminal]
 ----
-$ ./run-perf-profile-creator.sh -t /must-gather/must-gather.tar.gz -- --mcp-name=worker-cnf --reserved-cpu-count=2 --rt-kernel=true > my-performance-profile.yaml
+$ ./run-perf-profile-creator.sh -t /path-to-must-gather/must-gather.tar.gz -- --mcp-name=worker-cnf --reserved-cpu-count=1 --rt-kernel=true --split-reserved-cpus-across-numa=false --power-consumption-mode=ultra-low-latency --offlined-cpu-count=1 > my-performance-profile.yaml
 ----
 +
+* `--mcp-name=worker-cnf` specifies the `worker-=cnf` machine config pool.
+* `--reserved-cpu-count=1` specifies one reserved CPU.
+* `--rt-kernel=true` enables the real-time kernel.
+* `--split-reserved-cpus-across-numa=false` disables reserved CPUs splitting across NUMA nodes.
+* `--power-consumption-mode=ultra-low-latency` specifies minimal latency at the cost of increased power consumption.
+* `--offlined-cpu-count=1` specifies one offlined CPUs.
++
 [NOTE]
 ====
-The Performance Profile Creator arguments are shown in the Performance Profile Creator arguments table. The following arguments are required:
-
-* `reserved-cpu-count`
-* `mcp-name`
-* `rt-kernel`
-
 The `mcp-name` argument in this example is set to `worker-cnf` based on the output of the command `oc get mcp`. For {sno} use `--mcp-name=master`.
```

Comment:
@MarSik - regarding this note:
````
The `mcp-name` argument in this example is set to `worker-cnf` based on the output of the command `oc get mcp`. For {sno} use `--mcp-name=master`.
````

For single-node Openshift, we can call the MCP whatever we want, right? Just like in a regular cluster. Is it ok to delete this note or is there some complication with MCPs on SNO that I'm not getting?

Reply:
Nope, on SNO you have to use the `master` role.